content/w/sheamus.md

```diff
@@ -6,6 +6,8 @@ authors = ["Krzysztof Zych", "Sewi The Referee"]
 toclevel = 2
 [taxonomies]
 country = ["SCOTLAND"]
+[extra.gallery]
+1 = { path = "szejmus.webp", caption = "Seamus with his WWE United States Championship at a 2014 Smackdown show.", source = "Megan Elice Meadows / Wikipedia" }
```

Comment:
```suggestion
1 = { path = "szejmus.webp", caption = "Sheamus with his WWE United States Championship at a 2014 Smackdown show.", source = "Megan Elice Meadows / Wikipedia" }
```

Ideally the photo should be from one of the Polish shows, and with this one we can't tell.